### DISCUSSION TITLE
Not able load data

### DISCUSSION THREAD

**Discussion by srikannanb** [kaskada-ai/kaskada#208] [Q&A] [answered]
./kaskada-cli load --table Purchase --file-type csv --file-path file://Users/path/purchase.csv 
 INF starting load
WRN no client-id found, initiating request without passing client-id header.
 FTL  error="rpc error: code = Internal desc = internal error"

Getting this error while trying load the data.

Even I tried to purchase.parquet.Im getting the same error

**Comment by epinzur** [comment]
Hi, Can you provide some more details around the steps you were doing?  Have you successfully created the `Purchase` table?

Also if you could provide a copy of the logs from the Engine that would be helpful to figure out what the issue is.  Thanks.

**Comment by therapon** [comment, marked as answer]
Hi, this looks close to an issue we had yesterday and we pushed a fix to the instructions page #207

Could you try the following 

```
./kaskada-cli load \                
    --table Purchase \
    --file-typoe csv \
    --file-path file://${PWD}/purchase.csv
```

Some things to note here 

1. the `file:` portion needs 2 slashes 
2. the `file:` portion worked for me after giving the full path. the `${PWD}` will be replaced with the path to the current directory you are in at the moment.

**Comment by kerinin** [comment]
I just ran in to this in a trial. The problem is that this returns an internal error, so the client gives you practically no information about what's wrong - even if you look at the process logs it just says "failed getting source metadata". 

I think we should update the error messages to make it clear that the file you passed doesn't exist.

**Reply by therapon** [reply]
We have an issue #210 that includes a task for a better error message in the case when we cannot find the file at the location specified. 

We could split that task into its own issue.
